src/main/java/com/DGcreate/Main.java

```diff
@@ -14,6 +14,5 @@ public static void main(String[] args) {
         // hashset sorts on its own way
         HashSet<String> dict = new HashSet<>(stringsSorted);
         System.out.println(dict);
-        // 1 2 3 4 == 2 1 3 4 == 2 3 1 4 == 2 3 4 1
     }
 }
```

Comment:
Line 11 creates the sorted list of the words that we can use in programm, just like dictionary